packages/core/core/src/config.ts

```diff
@@ -183,7 +186,8 @@ export class Config {
       config: this,
       context: this.context,
       env: this.env,
-      version: this.version
+      version: this.version,
+      client: this.client
```

Comment:
It strikes me as a bit odd that all configs will have a client property when the config may have nothing to do with`client`.

Reply:
it was a side effect of removing the `options` object from the geth and parity packages, so they could be used as pure plugins. I expected this to be refactored at some point